packages/driver/src/cy/commands/misc.ts

```diff
@@ -14,15 +14,15 @@ export default (Commands, Cypress, cy, state) => {
   Commands.add('end', () => null)
   Commands.add('noop', (arg) => arg)
 
-  Commands.addQuery('log', (msg, ...args) => {
+  Commands.add('log', (msg, ...args) => {
```

Comment:
As per discussion in the docs PR, `.log()` should be a command and not a query.

Generally speaking, things that yield `null` don't make sense as queries, I just got overzealous and converted a few too many commands. :P

Reply:
I wasn't even looking for this change here. 👍🏻